draft-ietf-mls-protocol.md

```diff
@@ -2157,10 +2156,10 @@ The recipient of an UpdatePath processes it with the following steps:
        each hash incorporates all the non-blank nodes above it. The root node
        always has a zero-length hash for this value.
    * For nodes where a path secret was recovered in step 1 ("Compute the
-     updated path secrets"), compute and store the node's updated private key.
+     updated path secrets"), store the node's updated private key.
```

Comment:
The private key needs to computed from the path secret at some point.  Do you just mean for that to be implicit?